eng/native/init-vs-env.cmd

```diff
@@ -68,4 +68,4 @@ set "__VCBuildArch="
 
 :: Set CMakePath by evaluating the output from set-cmake-path.ps1.
 :: In case of a failure the output is "exit /b 1".
-for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy ByPass "%~dp0set-cmake-path.ps1"') do %%a
+for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy ByPass "& ""%~dp0set-cmake-path.ps1"""') do %%a
```

Comment:
Could we use `-File "%~dp0set-cmake-path.ps1"` instead?

Reply:
That works too and looks better.